packages/ra-ui-materialui/src/list/Datagrid.js

```diff
@@ -156,8 +156,8 @@ class Datagrid extends Component {
                                         currentSort={currentSort}
                                         field={field}
                                         isSorting={
-                                            field.props.source ===
-                                            currentSort.field
+                                            currentSort.field ===
```

Comment:
the || feels a bit weird here. Would you mind making the two conditions explicit?

```
isSorting={field.props.sortBy ? currentSort.field === field.props.sortBy : currentSort.field === field.props.source}
```

Reply:
I use the same style as seen in DatagridHeaderCell [here](https://github.com/marmelab/react-admin/blob/60fa0bdd43a11f4514cdd830d2d07811bde7f5dd/packages/ra-ui-materialui/src/list/DatagridHeaderCell.js#L42) and [here](https://github.com/marmelab/react-admin/blob/60fa0bdd43a11f4514cdd830d2d07811bde7f5dd/packages/ra-ui-materialui/src/list/DatagridHeaderCell.js#L56).
Should I change the || though?

Reply:
Oh, you're right. So for consistency's sake, let's keep it like that.